pkg/framework/aries/api/protocol.go

```diff
@@ -23,6 +24,7 @@ var ErrSvcNotFound = errors.New("service not found")
 // Provider interface for protocol ctx
 type Provider interface {
 	OutboundDispatcher() dispatcher.Outbound
```

Comment:
Do we still need `OutboundDispatcher in context`? can we deprecate it if not needed.

Reply:
It will be deprecated as soon as the messenger will be done!

Reply:
Do we have a TODO/Issue for this ? This looks to be a major refactor has we have been using Outbound dispatcher throughout the project.

Reply:
@soluchok fyi - may be you missed this comment.